src/Survey/ItechArt.Survey.WebApp/ViewModels/Mapping/UserProfile.cs

```diff
@@ -0,0 +1,13 @@
+﻿using AutoMapper;
+using ItechArt.Survey.DomainModel;
+
+namespace ItechArt.Survey.WebApp.ViewModels.Mapping;
+
+public class UserProfile : Profile
+{
+    public UserProfile()
+    {
+        CreateMap<ProfileViewModel, User>();
```

Comment:
1. add UserViewModel
2. configure mapping for it
3. Make ProfileViewModel contain UserViewModel property